src/Learners/Runners/Evaluator/x64JVMConfig.xml

```diff
@@ -0,0 +1,310 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+
+  <!--All common configuration elements shared between different test runs should be places inside this section-->
+  <Dictionary>
+    <!--Common settings for different implementations of the Matchbox model-->
+    <Element xsi:type="MatchboxRecommender" Id="Matchbox common">
```

Comment:
The experiment parameters exist in multiple places. This is something that we should avoid, because it leads to bugs in evaluation.

Reply:
Currently Evaluator needs all the parameters of an experiment to be passed in a single configuration file. This approach has its advantages and disadvantages, was chosen long ago, and changing it is not a part of this pull request. This means, that if I want to be able to run two experiments differing by just one parameter, I need two separate configuration files.

The reasons, I introduced this configuration and kept the old one are:
* In old configuration one of the Mahout runs failed, because 1 GB JVM heap was not enough
* Bigger heap sizes require 64 bit Java, so need `x64JVMConfig.xml` to be able to run everything successfully
* By default, 32 bit Java is installed (at least, it was for me), so I'd rather keep the old config as default

Reply:
If the new configuration is better, why do we need to keep the old one?

Reply:
As I said, it requires 64 bit Java, which is not what I've got installed by default. I also requires 6 GB of free RAM. So people may need to install a separate distribution of Java, and, possibly, an extra plank of RAM, in order to use it. That's why I think, it makes sense to keep the old one.

Reply:
Gotcha. Thanks for explaining.